api_specifications/application-api-0.1.0.yaml

```diff
@@ -310,6 +310,21 @@ components:
           example: Application 1 of the year 2024
         issuedBy:
           $ref: '#/components/schemas/IssuedBy'
+        applicationRaisedBy:
+          type: string
+          description: Application Raised by (Party Name of the advocate raising application)
+        representedBy:
+          type: string
+          description: Represented by (Advocate Name)
+        partyType:
+          type: string
+          description: Party Type (Complainant/Respondent)
+        reasonForApplication:
+          type: string
```

Comment:
There is a field - applicationType. Which is to link the master data for types of application. Is that not usable? it this a free flow text?

Reply:
I created this field to store reason, example  - Reason for Rescheduling, Reason for Extension.  For different application type we have to store the reason so I created this common field to store the reasonForApplication